esp/src/src-react/index.tsx

```diff
@@ -31,8 +31,14 @@ dojoConfig.urlInfo = {
 };
 dojoConfig.disableLegacyHashing = true;
 
-const store = userKeyValStore();
-store.getEx(ModernMode, { defaultValue: String(true) }).then(async modernMode => {
+const sessionStore = sessionKeyValStore();
+const userStore = userKeyValStore();
+Promise.all([
+    sessionStore.get(ModernMode),
+    userStore.getEx(ModernMode, { defaultValue: String(true) })
+]).then(([sessionModernMode, userModernMode]) => {
+    return sessionModernMode ?? userModernMode;
+}).then(async modernMode => {
```

Comment:
Common code, could be moved into `esp/src/src-react/hooks/platform.ts` as a regular function `fetchModernMode` or similar?